zgw_consumers/contrib/setup_configuration/models.py

```diff
@@ -0,0 +1,36 @@
+from django_setup_configuration.models import ConfigurationModel, DjangoModelRef
+from pydantic import Field
+
+from zgw_consumers.models import Service
+
+
+class SingleServiceConfigurationModel(ConfigurationModel):
+    # TODO these should probably be defined in simple_certmanager and referred to?
+    # client_certificate: FilePath = DjangoModelRef(Service, "client_certificate")
+    # server_certificate: FilePath = DjangoModelRef(Service, "server_certificate")
```

Comment:
I'm checking with Sjoerd if Dimpact uses these currently, if not this might be something for later

Reply:
Sjoerd told me that Dimpact doesn't use this currently, so I'm leaving this out of scope for now